Expensify/App
# Move all the compose meta data to be above the compose input

### Details
Some of the information used to be above the compose, and some of it below. All of them had inconsistent styling and bad practices for layout out the UI components.

I put the "user is typing" block at the end, since that's the one that has the biggest chance of flashing on and off. Putting it at the end makes it the least jarring so the position of the other elements don't change when it flashes on and off.

### Fixed Issues
$ https://github.com/Expensify/App/issues/9087

### Tests & QA
1. Open a chat
2. Go offline
3. Verify the offline message now appears above the compose input
4. Go back online
5. Compose a message with more than 60,000 characters (use copy/paste to do this easily)
6. Verify the comment exceeded meta data is displayed above the compose input
7. Open a second incognito browser and sign in as the other chat participant and start typing
8. Verify that the "user is typing" message is above the compose input
9. Open a third incognito browser and sign in as a third chat participant
10. Start a group chat with all three people
11. Quickly type something in the compose input in two of the browsers
12. Verify that in the third browser the "multiple people are typing" appears and it's displayed above the compose input
13. Verify that all the vertical and horizontal spacing looks OK


### Screenshots
**NOTE: For these screenshots, I hacked the code to display all pieces of meta data at once. This will not be possible to do in production because for example, you can't see "user is typing" when you're offline. **

Multiple users typing:

https://user-images.githubusercontent.com/1228807/169348542-71df1d65-2492-48d8-b75c-a567cd156dc9.mp4



#### Web

![2022-05-19_09-52-24](https://user-images.githubusercontent.com/1228807/169348746-6364cb7b-259e-47b7-9a18-aa5bdee7507c.png)

#### Mobile Web

![2022-05-19_10-07-11](https://user-images.githubusercontent.com/1228807/169348589-86681f8f-f94e-4205-9b95-cecaa1a3f972.png)

#### Desktop

![2022-05-19_09-58-48](https://user-images.githubusercontent.com/1228807/169348722-d9ca028d-558c-4d75-8c13-4039a9c33f14.png)

#### iOS

![2022-05-19_10-02-13](https://user-images.githubusercontent.com/1228807/169348646-54edcf88-3dde-4acd-bae0-636b915733f0.png)

#### Android

![2022-05-19_10-05-26](https://user-images.githubusercontent.com/1228807/169348615-2ccb95c1-3869-4d10-b10f-727c406374d1.png)
